src/modules/app/StoreActionType.js

```diff
@@ -97,11 +101,18 @@ export type MorphContentLanguageActionType = {|
   |}
 |}
 
+export type PushContentLanguageActionType = {|
+  type: 'PUSH_CONTENT_LANGUAGE', params: {|
```

Comment:
Use `SET_CONTENT_LANGUAGE` here. I chose `PUSH` to have a counter-action for `FETCH`.

So only name it `PUSH` if there is a corresponding `FETCH`